wcc-contentful/lib/wcc/contentful/model_singleton_methods.rb

```diff
@@ -12,9 +12,9 @@ module WCC::Contentful::ModelSingletonMethods
   #   WCC::Contentful::Model::Page.find(id)
   def find(id, options: nil)
     options ||= {}
-    store = store(options[:preview])
+    store = options[:preview] ? services.preview_store : services.store
```

Comment:
have we verified preview_store / services are working properly in these multi-tenant changes?

Reply:
There is one set of integration tests asserting that preview works for `WCC::Contentful::Model`, but none specifically testing that a `ModelAPI` included in some other namespace properly sets up the preview store.  That's probably one worth having and I'll make it happen.